text.go

```diff
@@ -30,12 +30,33 @@ const (
 	TextWrapWord
 )
 
+const (
+	// DefaultTabWidth is the default width in spaces
```

Comment:
API introduced needs a since header:
```
//
// Since: 2.1
```

However - I don't think this API is necessary. If the defaulting was handled in the renderer and widget packages we could hide it in an `internal` package until we decide if this detail should be exposed...